lib/pure/ospaths.nim

```diff
@@ -423,12 +423,22 @@ proc isAbsolute*(path: string): bool {.rtl, noSideEffect, extern: "nos$1".} =
   ## Checks whether a given `path` is absolute.
   ##
   ## On Windows, network paths are considered absolute too.
+  runnableExamples:
+    doAssert "".isAbsolute.not
+    doAssert nil.isAbsolute.not
```

Comment:
There soon will be no ``nil`` for strings, don't write tests covering it.

Reply:
done